pkg/controller/controller_utils.go

```diff
@@ -449,6 +449,8 @@ type PodControlInterface interface {
 	CreatePods(namespace string, template *v1.PodTemplateSpec, object runtime.Object) error
 	// CreatePodsWithControllerRef creates new pods according to the spec, and sets object as the pod's controller.
 	CreatePodsWithControllerRef(namespace string, template *v1.PodTemplateSpec, object runtime.Object, controllerRef *metav1.OwnerReference) error
+	// CreatePodsWithControllerRefAndPrefix creates new pods according to the spec, and sets prefix to pod's generateName.
```

Comment:
the docs are confusing... what prefix is set?

Reply:
Update method name to `CreatePodsWithControllerRefAndGenerateName `.
Thanks for reviewing : )